plugins/fused-ops-and-kernels/src/fms_acceleration_foak/models/granite.py

```diff
@@ -40,6 +41,7 @@ def get_mp_rules(base_type: str):
     try:
         # Third Party
         from transformers.models.granite.modeling_granite import (  # pylint: disable=import-outside-toplevel
+            GraniteForCausalLM,
```

Comment:
I'm guessing granite here refers to teh GraniteForCausalLM models and the granite with llama architecture models would be handled within the llama.py correct? I'm not sure if the liger kernels work for any of the granite models, do you know?

Reply:
thats the problem with this solution, because we copy and paste the forward functions wholesale. so if the forwards are different this doesnt work